content/_jobs/2021-04-21-dev-piaf.md

```diff
@@ -0,0 +1,64 @@
+---
+roles: Full-Stack expérimenté + appétence infra
+title: "L'équipe Piaf recrute un Full-Stack expérimenté avec une appétence pour l'infra"
```

Comment:
@guillim Pour le titre, il faudrait que la formulation soit inclusive ou épicène.

Je crois que "title" n'est pas utilisé d'ailleurs.

Reply:
True, I just changed it. Let me know if you see any other inconsistencies